### PR TITLE
Vectorize

### PR DESCRIPTION
**Pull request by drewbo**

### CODE REVIEW

File: vectorize.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+var _ = require('lodash')
+var distance = require('turf-line-distance')
+var simplify = require('turf-simplify')
+
+var input = JSON.parse(fs.readFileSync(path.join(__dirname, argv._[0])))
+input = normalize(input)
+input.features = _.flatten(input.features.map(f => flatten(f)))
+
+// arguments given as filename, then z y x
+var tile = argv._.slice(1, 4).reverse()
```
Comment (anandthakker) [inline vectorize.js]: Could we go with either `x y z` or `z x y`?  (Slight personal pref for the latter, but only slight)

File: vectorize.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+var normalize = require('geojson-normalize')
+var _ = require('lodash')
+var distance = require('turf-line-distance')
+var simplify = require('turf-simplify')
+
+var input = JSON.parse(fs.readFileSync(path.join(__dirname, argv._[0])))
+input = normalize(input)
+input.features = _.flatten(input.features.map(f => flatten(f)))
+
+// arguments given as filename, then z y x
```
Comment (anandthakker) [inline vectorize.js]: Add to note: ", then distance threshold as ratio of image width"

**Review by anandthakker — CHANGES_REQUESTED** [review]
Looks good, just a couple comments

File: vectorize.js (hunk 1 above)
Comment (drewbo) [reply vectorize.js]: Let's do `x y z` then because it's how `tilebelt` accepts it

Comment (anandthakker) [reply vectorize.js]: 👍

**Pull request merged**